checks/zk_quorum.go

```diff
@@ -0,0 +1,129 @@
+package checks
+
+import (
+    "bufio"
+    "fmt"
+	"time"
+    "strings"
+    "regexp"
+    "os"
+
+	"github.com/CiscoCloud/distributive/chkutil"
+	"github.com/CiscoCloud/distributive/errutil"
+	"github.com/samuel/go-zookeeper/zk"
+)
+
+/*
+#### ZooKeeperQuorum
+Description: Are these Zookeeper servers responding to "ruok" requests?
+Parameters:
+- Timeout (time.Duration): Timeout for server response
+- Config file: file with zk config, where all nodes listed
+Example parameters:
+- "5s", "20ms", "2h"
+- "/etc/zookeeper/conf/zoo.cfg"
+*/
+type ZooKeeperQuorum struct {
+	timeout time.Duration
+    Config  string
+}
+
+func init() { 
+    chkutil.Register("ZooKeeperQuorum", func() chkutil.Check {
+        return &ZooKeeperQuorum{
+            Config: "/etc/zookeeper/conf/zoo.cfg",  /* default value, only for new k/v arg parsing */
+        }
+    })
+}
+
+func (chk ZooKeeperQuorum) New(params []string) (chkutil.Check, error) {
+	if len(params) != 2 {
```

Comment:
Defaults will work only with new K/V arg parsing, it just some residue from ruther development